src/AsyncGenerator.yml

```diff
@@ -212,6 +219,8 @@
       projectFiles:
       - fileName: LinqExtensionMethods.cs
         projectName: NHibernate
+      - fileName: AsyncExtensions.cs
```

Comment:
This will not work even with version `0.21.0` as `AsyncExtensions` class does not exist for `net6.0` which the `NHibernate.Test` project targets. You need to add extension methods for both `ISession` and `IStatelessSession` in case it is not desired to have a breaking change and then registered them like this:
```
    asyncExtensionMethods:
      projectFiles:
      - fileName: LinqExtensionMethods.cs
        projectName: NHibernate
      assemblyTypes:
      - fullTypeName: NHibernate.SessionExtensions
        assemblyName: NHibernate
      - fullTypeName: NHibernate.StatelessSessionExtensions
        assemblyName: NHibernate
```
With version AsyncGenerator 0.21.0 onwards, it is recommended that `assemblyTypes` option is used instead of `projectFiles` when referencing types from a different project, otherwise it is possible that the extension methods won't be used.

Reply:
> This will not work even with version 0.21.0 as AsyncExtensions class does not exist for net6.0

Yeah, this was my mistake, I've put it into incorrect place at first, I just did not care to clear it.